tools/MultiPEN/MultiPEN-cross-validation.xml

```diff
@@ -0,0 +1,49 @@
+<tool id="multipen-cross-validation" name="crossValidation" version="0.0.2">
+    <description>feature selection from expression levels</description>
+    <requirements>
+        <requirement type="package" version="0.0.2">MultiPEN</requirement>
+    </requirements>
+    <stdio>
+        <exit_code range="1:" level="fatal" />
+    </stdio>
+    <command>
+<![CDATA[
+    run_MultiPEN.sh cross_validation ./ "$expr_levels" "$interaction_network" "$classes" $lambda $n_folds $max_iter &&
+    mv MultiPEN-Rankings_lambda${lambda}.txt Rankings.txt &&
+    mv MultiPEN_Rankings_lambda${lambda}.mat Rankings.mat &&
+    mv MultiPEN-vts_lambda${lambda}.txt vts.txt &&
+    mv MultiPEN-Rankings_lambda${lambda}_genes-higher-in-cases.txt Rankings-higherInCases.txt &&
+    mv MultiPEN-Rankings_lambda${lambda}_genes-higher-in-control.txt Rankings-higherInControl.txt &&
+    
+]]>
+    </command>
+    <inputs>
+        <param name="expr_levels" type="data" format="tabular" label="Expression levels for features" help="Gene expression and/or metabolite levels" />
+        <param name="interaction_network" type="data" format="tabular" label="Interaction Network" help="Matrix with the interactions, each interaction is of the form: [node, node, interaction_weight]" />
+        <param name="classes" type="data" format="txt" label="Classes for the samples" help="1 for case or 0 for control" />
+        <param name="lambda" type="float" value="" label="Lambda for the optimisation function" help="" />
+        <param name="n_folds" type="integer" value="" label="N-folds for cross validation" help="" />
+        <param name="max_iter" type="integer" value="" label="Maximum number of iterations for optimisation" help="Optional parameter, by default this is set to 100" />
```

Comment:
If this is optional, can you add `optional="true"` (usually we put it between `value` and `label`)?